resqpy/well/well_object_funcs.py

```diff
@@ -408,6 +408,8 @@ def add_blocked_wells_from_wellspec(model, grid, wellspec_file):
 
     log.info(f'{count} blocked wells created based on wellspec file: {wellspec_file}')
 
+    return count  # TODO: Kadija, confirm that this return statement should be here. You added this!
```

Comment:
The code is consistent with the docstring, and will be a harmless change from no value being returned, so I think the TODO can be removed.